docs/docs/getting-started/install.md

```diff
@@ -15,12 +15,90 @@ Make sure the following requirements are met:
 
 ## Install the Constellation CLI
 
-Download the CLI executable from the [release page](https://github.com/edgelesssys/constellation/releases). Move the downloaded file to a directory in your `PATH` (default: `/usr/local/bin`) and make it executable by entering `chmod u+x constellation` in your terminal. 
+Download the CLI executable from [GitHub](https://github.com/edgelesssys/constellation/releases). Use `install` to copy the downloaded file to your `PATH` (default: `/usr/local/bin`) and automatically make it executable.
 
 :::note
-Edgeless Systems uses [sigstore](https://www.sigstore.dev/) to sign each release of the CLI. You may want to [verify the signature](../workflows/verify-cli.md) before you use the CLI.
+Edgeless Systems uses [sigstore](https://www.sigstore.dev/) to sign each release of the CLI. You may want to [verify the signature](../workflows/verify-cli.md) before you install and use the CLI.
 :::
 
+You can use the following commands for this:
```

Comment:
With the new instructions, this is all quite repetitive. I'd just drop the note and shorten all this to
```markdown
The CLI executable is available at [GitHub](https://github.com/edgelesssys/constellation/releases).
Install it with the following commands:
```